scripts/sshzobelisk

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+usage()
+{
+    echo "USAGE: $0 <USERNAME ON ZOBELISK> <USER ID ON ZOBELSIK>"
+    exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+LOCAL_USER=$USER
+SERVER_USERNAME=$1
+SERVER_UID=$2
+LOCAL_REPO=$(realpath $(dirname $BASH_SOURCE)/../)
+ZOBELISK_IP=192.168.37.176
+IP=`hostname -I | head -n1 | awk '{print $1;}'`
+
+read -n 1 -p "WARNING: This script requires you to run an ssh server on your computer while you are on the development server. Some people consider this to be insecure, but this is the price of using GUI text editors with a remote system. If you do not want to run an ssh server, kill this script now. There is an alternative script, sshzobelisk_safe which does not support file sharing between the machines, but does not require openssh-server to be run on your machine. The script will automatically close the ssh server upon exiting. Press Ctrl + c if you do not consent. Otherwise, press any key to continue." PLACEHOLDER
```

Comment:
Cool idea to have a version that doesn't require an ssh server, but it is more appropriate to make this a flag or environment variable in this script rather than having a near-duplicate script